pywbem_mock/_baseprovider.py

```diff
@@ -286,6 +284,14 @@ def add_namespace(self, namespace):
         if namespace is None:
             raise ValueError("Namespace argument must not be None")
 
+        # Cannot add more than one of the possible interop namespace names
+        for name in self.interop_namespace_names:
+            if name in self.namespaces:
+                raise CIMError(
+                    CIM_ERR_ALREADY_EXISTS,
+                    _format("An interop Namespace {0!A} already exists in the "
+                            "CIM repository. {} cannot be added. ",
+                            name, namespace))
```

Comment:
This second use of CIM_ERR_ALREADY_EXISTS should be documented in the Raises section of the docstring.

Reply:
DONE